addons/account/models/account_move.py

```diff
@@ -871,27 +873,32 @@ def _compute_name(self):
             if move.state == 'cancel':
                 continue
 
-            move_has_name = move.name and move.name != '/'
-            if move_has_name or move.state != 'posted':
-                if not move.posted_before and not move._sequence_matches_date():
-                    if move._get_last_sequence():
-                        # The name does not match the date and the move is not the first in the period:
-                        # Reset to draft
-                        move.name = False
-                        continue
-                else:
-                    if move_has_name and move.posted_before or not move_has_name and move._get_last_sequence():
-                        # The move either
-                        # - has a name and was posted before, or
-                        # - doesn't have a name, but is not the first in the period
-                        # so we don't recompute the name
-                        continue
-            if move.date and (not move_has_name or not move._sequence_matches_date()):
+            move_has_name = move.name and move.name != "/"
```

Comment:
I feel like there is a lot of useless diff here (changing double/single quotes, moving around `if` blocks whose order shouldn't matter)
It makes the diff harder to read and understand. (and all the issues linked to potential conflicts)

Am I missing something?